documentation/en/api-v0-methods-miner.md

```diff
@@ -345,6 +349,46 @@ Inputs:
 
 Response: `{}`
 
+## Dagstore
+
+
+### DagstoreGC
+DagstoreGC runs garbage collection on the DAG store.
+
+
+Perms: admin
+
+Inputs: `null`
+
+Response: `null`
+
+### DagstoreInitializeShard
+DagstoreInitializeShard initializes an uninitialized shard by acquiring
+it and releasing it as soon as it's ready.
```

Comment:
I think it's not that clear why you would want to use this command if you don't know how the DAG store works internally. Could we add some context to this description? (and maybe the description in the CLI help)

Reply:
Ack, I didn't feel happy writing this. Will revise.